open-apparel-registry/open-apparel-registry
# Update to Dedupe 2.0.13

## Overview
this PR separates out the work upgrading to dedupe 2.0.13 from the work rearchitecting the dedupe process. The commits for this PR have been copied from from #1309 using `git cherry-pick`.

Connects #1955 

## Demo
<img width="1206" alt="Screen Shot 2022-07-13 at 17 47 04" src="https://user-images.githubusercontent.com/12550593/178842581-444f0506-4f34-470a-a27f-616d307fdf39.png">

## Notes
#1309 will be rebased off `ogr/develop` after this branch is merged

## Testing Instructions
* run `./scripts/server` to start the site
* through the [Contribute page](http://localhost:6543/contribute), upload [this](https://github.com/open-apparel-registry/open-apparel-registry/files/9106334/azavea_osh.csv) csv
* after the list is uploaded, make note of the list number at the end of the URL bar, in this case `http://localhost:6543/lists/18`
* at the command line in the project directory, run the command `./tools/batch_process XX`, where XX is the list number from the previous step

A Successful run should look like this:
```[+] Running 1/0
 ⠿ Container open-apparel-registry-database-1  Running                                                                                   0.0s
/usr/local/lib/python3.7/site-packages/thefuzz/fuzz.py:11: UserWarning: Using slow pure-python SequenceMatcher. Install python-Levenshtein to remove this warning
  warnings.warn('Using slow pure-python SequenceMatcher. Install python-Levenshtein to remove this warning')
parse: 1 successes
[+] Running 1/0
 ⠿ Container open-apparel-registry-database-1  Running                                                                                   0.0s
/usr/local/lib/python3.7/site-packages/thefuzz/fuzz.py:11: UserWarning: Using slow pure-python SequenceMatcher. Install python-Levenshtein to remove this warning
  warnings.warn('Using slow pure-python SequenceMatcher. Install python-Levenshtein to remove this warning')
geocode: 1 successes
[+] Running 1/0
 ⠿ Container open-apparel-registry-database-1  Running                                                                                   0.0s
/usr/local/lib/python3.7/site-packages/thefuzz/fuzz.py:11: UserWarning: Using slow pure-python SequenceMatcher. Install python-Levenshtein to remove this warning
  warnings.warn('Using slow pure-python SequenceMatcher. Install python-Levenshtein to remove this warning')
match: 1 successes
```


## Checklist
